infra/cifuzz/actions/build_fuzzers/action.yml

```diff
@@ -38,3 +38,4 @@ runs:
     LOW_DISK_SPACE: 'True'
     BAD_BUILD_CHECK: ${{ inputs.bad-build-check }}
     CIFUZZ_DEBUG: 'True'
+    CI_SYSTEM: 'github'
```

Comment:
Maybe CFL_PLATFORM or CFL_ENV to be less generic?

Reply:
I thought we didn't want to do `CFL_` prefix. Maybe we should do it for really generic variables only then?

Reply:
Yep that's my thought. Add CFL_ for very generic ones like these.

Reply:
:-( I don't like the inconsistency though.